codebases/index.md

```diff
@@ -0,0 +1,37 @@
+
+# Where we work: our codebases
+
+The Foundation for Public Code supports ambitious communities and their codebases to scale effectively, including smoother collaboration and shared governance.
+
+Codebases are organized by [codebase maturity](https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html).
```

Comment:
```suggestion
We organize codebases into different phases according to their [maturity](https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html).
```

Reply:
I'm not going to merge your change. This is because:
- I'd like to avoid adding extra words above the fold - our goal is for people to jump as quickly as possible into looking at the codebases
- We already have further changes planned to this page as part of #64